### DISCUSSION TITLE
[Support]: Unable to pull Docker image on QNAP

### DISCUSSION THREAD

**Discussion by feltroncq** [blakeblackshear/frigate#15300] [General Support]
### Describe the problem you are having

I've been struggling with Certificates from an unknown source, but have seemingly resolved that and have successfully pulled several images into Container Station which I am also seeing in Portainer.
When I try to follow your directions at 
https://docs.frigate.video/frigate/installation#qnap-nas
I get as far as 
`docker pull ghcr.io/blakeblackshear/frigate:stable`
but I am continually getting an error message saying 
Error response from daemon: manifest for ghcr.io/blakeblackshear/frigate:stable not found
I have been messing with this for hours now, with no success. I have even tried downloading the image and importing to Portainer, with similar results

### Version

0.14.1

### What browser(s) are you using?

Google Chrome Version 131.0.6778.86 (Official Build) (64-bit)

### Frigate config file

```yaml
Not built yet. Still trying to pull image into Docker
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Install method

Docker CLI

### docker-compose file or Docker CLI command

```yaml
docker pull ghcr.io/blakeblackshear/frigate:stable
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

N/A

### Screenshots of the Frigate UI's System metrics pages

N/A

**Comment by hawkeye217** [comment]
None of the maintainers have a QNAP, so perhaps someone from the community can help with your particular issue.

However, I can tell you that you may struggle to run Frigate 0.14 with Container Station, which I believe is still limited to a shm-size of only 64mb. Frigate 0.14 requires a fair amount of shared memory. This has been improved in Frigate 0.15 (currently in beta), but even then, 64mb may only be sufficient for one or perhaps two cameras.

**Reply by feltroncq** [reply]
There does seem to be a workaroud for the shm-size issue in that link I posted. I have Frigate working on another platform, but just inherited the QNAP and wanted to see if I can do anything with it. If I can't, it's not biggy. Just frustrating that the Frigate images wont pull. Weird!
